ansible/roles/dokuwiki/tasks/main.yml

```diff
@@ -25,7 +25,19 @@
     - prerequisites
 
 - name: Download DokuWiki stable source code
-  shell: "wget https://download.dokuwiki.org/src/dokuwiki/dokuwiki-stable.tgz -O - | tar -xzvf - -C /opt/dokuwiki --strip 1"
+  shell: "wget https://download.dokuwiki.org/src/dokuwiki/dokuwiki-stable.tgz -O - | tar -xzf - -C /opt/dokuwiki --strip 1"
```

Comment:
`tar -xzf -` is the same as `tar xz` ;) If you skip the `f` flag then it will read from stdin.

Reply:
Also we should probably pin the dokuwiki version since we're poking around in its internals.